tests/end_to_end/tests.py

```diff
@@ -1290,6 +1288,17 @@ def step_client_delete(self):
                     found_client = get_client(self.config, client, ssn)
                     assert len(found_client) == 0
 
+    def step_make_owner(self):
+        member = 'DEV:COM:222'
+        with XRDSSTTest() as app:
+            client_controller = ClientController()
+            client_controller.app = app
+            configuration = client_controller.create_api_config(self.config["security_server"][0], self.config)
```

Comment:
It seems you are performing the operation for all security servers, but creating the api key only for the first one, it will probably not work that way